controllers/auth.js

```diff
@@ -42,7 +42,7 @@ const githubAuth = (req, res, next) => {
         sameSite: "lax",
       });
 
-      if (incompleteUserDetails) authRedirectionUrl = "https://my.realdevsquad.com/new-signup";
+      // if (incompleteUserDetails) authRedirectionUrl = "https://my.realdevsquad.com/new-signup";
```

Comment:
Should we keep dead code around?

Reply:
have reverted this change as well, had done it while setting up the local server on the machine